content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -2013,18 +2056,31 @@ The name of the `fields` argument on the `@@unique` attribute can be omitted:
 @@unique(fields: [title, author], name: "titleAuthor")
 ```
 
```

Comment:
We should mention here that the length and sort args go as args into the field reference and are not their own top level arguments on compounds.

Reply:
I've added an example snippet to demonstrate this - is this correct?